app/models/model_events/plan_year.rb

```diff
@@ -51,9 +55,9 @@ def notify_on_save
           is_open_enrollment_began = true
         end
 
-        if enrolled? || renewing_enrolled?
-          is_open_enrollment_ended = true
-        end
+        # if enrolled? || renewing_enrolled?
```

Comment:
Remove commented out code if not necessary.

Reply:
@raghuramg added those lines at the time of demo. I do not know the reason why they added. will remove.